README.md

```diff
@@ -50,162 +43,112 @@ source and public models in popular formats such as TensorFlow, ONNX, PaddlePadd
 * [Frontends](./src/frontends) - contains available OpenVINO frontends that allow reading models from the native framework format.
 * [OpenVINO Model Converter (OVC)] - is a cross-platform command-line tool that facilitates the transition between training and deployment environments, and adjusts deep learning models for optimal execution on end-point target devices.
 * [Samples] - applications in C, C++ and Python languages that show basic OpenVINO use cases.
+</details>
```

Comment:
Propose to remove a section `Components`. this is mostly for developers. For developer we should have a link here to developer documentation. And it should be enough

Reply:
Developer documentation should contain a list of components and more detail info